gulpfile.js

```diff
@@ -122,7 +122,15 @@ gulp.task('gitRevisions', function(cb) {
     if (!err) {
       revisionInfo.revision = _.trim(data);
     }
-    if(revisionInfo.repo && revisionInfo.revision) {
+    else {
+      cb();
+    }
+  }).raw(["branch"], (err, data) =>
+  {
```

Comment:
The opening brace should be put in the same line as the arrow function notation.